services/provider/client/client.go

```diff
@@ -133,3 +133,54 @@ func (cc *OCSProviderClient) AcknowledgeOnboarding(ctx context.Context, consumer
 
 	return cc.Client.AcknowledgeOnboarding(apiCtx, req)
 }
+
+func (cc *OCSProviderClient) ProvisionStorageClassClaim(ctx context.Context, consumerUUID, claimUUID string, storageType pb.ProvisionStorageClassClaimRequest_StorageType) (*pb.ProvisionStorageClassClaimResponse, error) {
+	if cc.Client == nil || cc.clientConn == nil {
+		return nil, fmt.Errorf("provider client is closed")
+	}
+
+	req := &pb.ProvisionStorageClassClaimRequest{
+		StorageConsumerUUID:   consumerUUID,
+		StorageClassClaimUUID: claimUUID,
+		StorageType:           storageType,
+	}
+
+	apiCtx, cancel := context.WithTimeout(ctx, cc.timeout)
+	defer cancel()
+
+	return cc.Client.ProvisionStorageClassClaim(apiCtx, req)
```

Comment:
Are we getting the remote claim UID in the response?

Reply:
Yes will add it to the Response.